### PR TITLE
Tiny JS World

### PR DESCRIPTION
**Pull request by hisbvdis**
### Tiny JS World

[Demo](https://hisbvdis.github.io/kottans-frontend/pages/a-tiny-JS-world/) |
[Code base](https://github.com/hisbvdis/kottans-frontend/tree/main/pages/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
@hisbvdis why `reduce`?

**Comment by hisbvdis** [comment]
> @hisbvdis why `reduce`?

Собирал с помощью него в строку результаты вызовов.
Но, наверное, так действительно излишне сложно.
Заменил на `map()`

File: submissions/hisbvdis/tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  "legs",
+  "hands",
+  "saying",
+];
+
+inhabitants
+  .map((inhabitant) =>
+    inhabitantsProps.map((prop) => inhabitant[prop]).join("; ")
+  )
+  .forEach((inhabitant) => print(inhabitant));
```
Comment (OleksiyRudenko) [inline submissions/hisbvdis/tiny-js-world/index.js:84]: Need to fix this
image

Let your [IDE do the dirty job for you](https://github.com/OleksiyRudenko/dev-env-git#ide-minimal-tweak).

**Review by OleksiyRudenko — APPROVED** [review]
@hisbvdis great job!

File: submissions/hisbvdis/tiny-js-world/index.js (hunk 1 above)
Comment (hisbvdis) [reply submissions/hisbvdis/tiny-js-world/index.js:84]: Спасибо

**Pull request merged**